client/app/authenticators/zap-api-authenticator.js

```diff
@@ -28,7 +28,7 @@ export default class ZAPAuthenticator extends BaseAuthenticator {
   // a ZAP Token. This request will fail if that cookie doesn't exist
   // https://github.com/simplabs/ember-simple-auth/blob/master/guides/managing-current-user.md#using-a-dedicated-endpoint
   async _fetchUserObject() {
```

Comment:
should this be named _fetchContactObject() instead?

Reply:
yessir! good catch